src/endpoints/v1.es6.js

```diff
@@ -1273,35 +1273,36 @@ class APIv1Endpoint {
   get wiki () {
     return bind({
       buildOptions: function(options) {
-        var uri = options.origin;
+        let uri = options.origin;
         if (options.subreddit) {
           uri += `/r/${options.subreddit}/wiki/${options.path}.json`;
         } else {
           uri += `/wiki/${options.path}.json`;
         }
 
-        return { uri, options };
+        return { uri, reqOptions };
```

Comment:
`reqOptions` is not defined. Also, since everything returns `{ uri, options }` it may be better to change the name of the function argument instead, so you can return `{ uri, options }` like everything else.